xcm/pallet-xcm-benchmarks/src/fungible/mock.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
not super relevant yet since this PR is only about the `fungible`, but once a `fungibles` has been added as well, it would be sad to see this mock almost entirely duplicated. I think one mock runtime that implements both `xcm_benchamrk::fungible::Config` and `xcm_benchamrk::fungibles::Config` should work fine.